swagger.json

```diff
@@ -2117,6 +2117,170 @@
                 ]
             }
         },
+        "/organizers/{cdbid}/website": {
+            "put": {
+                "summary": "Update the website of a specific organizer",
+                "operationId": "updateWebsite",
+                "produces": [
+                    "application/json"
+                ],
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/cdbid"
+                    },
+                    {
+                        "name": "body",
+                        "in": "body",
+                        "description": "An object containing the website of an organizer.",
+                        "required": true,
+                        "schema": {
+                            "$ref": "#/definitions/Website"
+                        }
+                    }
+                ],
+                "responses": {
+                    "202": {
+                        "$ref": "#/responses/CommandQueued"
+                    },
+                    "401": {
+                        "$ref": "#/responses/Unauthorized"
+                    },
+                    "403": {
+                        "$ref": "#/responses/Forbidden"
+                    },
+                    "400": {
+                        "$ref": "#/responses/APIProblem"
+                    }
+                },
+                "tags": [
+                    "Organizer",
+                    "Website"
+                ]
+            }
+        },
+        "/organizers/{cdbid}/title": {
```

Comment:
Volgens de huidige Swagger docs noemt dit property eigenlijk "name" in de JSON-LD van zowel organizer als event & place. Misschien dan beter hier ook "name" gebruiken ipv "title" om consistent te blijven?